### PR TITLE
use new otp-ui update to use long name

### PR DESCRIPTION
**Pull request by daniel-heppner-ibigroup**
Corresponding update to https://github.com/opentripplanner/otp-ui/pull/708.

### CODE REVIEW

**Comment by daniel-heppner-ibigroup** [comment]
This can merge with one approval. Please merge after you approve

File: lib/components/map/connected-transit-vehicle-overlay.js [hunk 1]
```diff
@@ -138,8 +138,8 @@ const mapStateToProps = (state) => {
           ? '#' + route.color
           : route?.color || '#5A5A5A'
       // Try to populate this attribute, which is required for the vehicle popup to appear.
-      vehicle.routeShortName =
-        vehicle.routeShortName || route?.shortName || true
+      vehicle.routeShortName = vehicle.routeShortName || route?.shortName
+      vehicle.routeLongName = vehicle.routeLongName || route?.longName
```
Comment (miles-grant-ibigroup) [inline lib/components/map/connected-transit-vehicle-overlay.js:142]:
```suggestion
      vehicle.routeLongName = vehicle?.routeLongName || route?.longName
```
Should this be optional?

**Review by miles-grant-ibigroup — APPROVED** [review]
One quick question, otherwise approved providing otp-ui version update is in this PR

**Comment by miles-grant-ibigroup** [comment]
Sorry @daniel-heppner-ibigroup I messed up my branch selection and this is now contains a bunch of approved code

**Comment by miles-grant-ibigroup** [comment]
Since this is also a piggyback, let's merge this into the other PR and then update the package there

**Pull request merged**